File: 3227-FindMissingAndRepeatedValues/3227-FindMissingAndRepeatedValues.java
// Last updated: 31/07/2025, 21:06:02
class Solution {
    public int[] findMissingAndRepeatedValues(int[][] grid) {
        int n=grid.length;
        int[] freq=new int[n*n + 1];
        for(int i=0;i<n;i++)
        {
            for(int j=0;j<grid[0].length;j++)
            {
                freq[grid[i][j]]+=1;
            }
        }
        int[] arr=new int[2];
        for(int i=1;i<freq.length;i++)
        {
            if(freq[i]==0)
            {
                arr[1]=i;
            }
            if(freq[i]==2)
            {
                arr[0]=i;
            }
        }
        return arr;
    }
}

File: 3227-FindMissingAndRepeatedValues/3227-FindMissingAndRepeatedValues.py
# Last updated: 31/07/2025, 21:06:08
class Solution:
    def findMissingAndRepeatedValues(self, grid: List[List[int]]) -> List[int]:
        newgrid=[item for j in grid for item in j ] 
        newgrid.sort()
        for i in newgrid:
            if newgrid.count(i)==2:
                c=i
                break
        s=list(set(newgrid))
        s.append(0)
        ind=1
        k=0
        while True:
            if ind==s[k]:
                k+=1
                ind+=1
            else:
                break         
        n=[]
        n.append(c)
        n.append(ind)
        return n            
